public/app.css

```diff
@@ -68,17 +73,43 @@ body {
   padding-right: 20px;
   padding-left: 20px;
 }
-.nav-sidebar > .active > a,
-.nav-sidebar > .active > a:hover,
-.nav-sidebar > .active > a:focus {
-  /*color: #fff;*/
-  /*background-color: #dddddd;*/
+.nav-sidebar li.unimplemented{
```

Comment:
these `*li.unimplemented*` rules grey out unimplemented rules on the sidebar and create tooltips that appear when you mouse over them